packages/dds/tree/src/test/feature-libraries/modular-schema/comparison.spec.ts

```diff
@@ -345,22 +300,23 @@ describe("Schema Comparison", () => {
 			const name = brand<TreeNodeSchemaIdentifier>("testTree");
 			const root = fieldSchema(FieldKinds.optional, [name]);
 
-			const testTree1 = new ObjectNodeStoredSchema(
-				new Map([
-					[brand("x"), fieldSchema(FieldKinds.optional, [emptyTree.name])],
-					[brand("y"), fieldSchema(FieldKinds.optional, [emptyTree.name])],
-				]),
-			);
+			const testTree1 = nodeWithOptionalXAndYFields;
 			const testTree2 = new ObjectNodeStoredSchema(
 				new Map([[brand("x"), fieldSchema(FieldKinds.sequence, [emptyTree.name])]]),
 			);
 			const repo1 = new TreeStoredSchemaRepository({
 				rootFieldSchema: root,
-				nodeSchema: new Map([[name, testTree1]]),
+				nodeSchema: new Map([
```

Comment:
this test used to reference but not include emptyTree, which make it not representative of real use.